src/main/java/reposense/git/GitVersion.java

```diff
@@ -4,19 +4,20 @@
 
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
  * Contains git version related functionalities.
  * Git version is responsible for finding out the version of git the user of RepoSense is running.
  */
 public class GitVersion {
-    public static final String FINDING_PREVIOUS_AUTHORS_INVALID_VERSION_WARNING_MESSAGE =
```

Comment:
I'm not sure why we are moving this.

Reply:
My reasoning was that we should pair the message constant with the location where the logging is actually done. So either we log and store the constant in `GitVersion`, or we log and store in `RepoSense`. 

I moved it to `RepoSense` because I wanted to the method to be without side effects so that whenever we call the `isGitVersionSufficient...` method can be called without filling up the log.

Reply:
Nice

Reply:
Thanks for the clarification. I had the same question.